electron/electron.livepeer.js

```diff
@@ -18,9 +18,10 @@ const startLivepeer = (sender) => {
             global.sharedObj.livepeerProc = 'local';
             log.info('LivePeer is already running.');
         } else if (global.sharedObj.livepeerProc == null) {
-            const args = [];
+            const args = [
+                '-monitor', true,
+                '-monitorhost', 'http://viz.livepeer.org:8081/metrics'];
```

Comment:
@buildog - I have a quick question.  Trying to add these args so the nodes will show up on http://viz.livepeer.org:8081

For some reason, the url becomes a different string (metrics.livepeer.org) when it gets into the node...

```
(node:83818) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Invalid template for Menu
(node:83818) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[17:02:21:0045] [info] Livepeer running on port 8935
[17:02:21:0122] [info] stderr: I0817 17:02:21.101137   83821 livepeer.go:118] Setting monitor endpoint: metrics.livepeer.org
I0817 17:02:21.103104   83821 network_node.go:71] Created node: 12209433a695c8bf34ef6a40863cfe7ed64266d876176aee13732293b63ba1637fd2 at [/ip4/127.0.0.1/tcp/15000 /ip6/::1/tcp/15000 /ip4/10.30.105.34/tcp/15000]
I0817 17:02:21.103251   83821 basic_network.go:192]

Setting up protocol: /livepeer_video/0.0.1
```

Do you know what could cause this?